canvasapi/account.py

```diff
@@ -1702,14 +1804,66 @@ def update_role(self, role, **kwargs):
 @python_2_unicode_compatible
 class AccountNotification(CanvasObject):
     def __str__(self):  # pragma: no cover
-        return "{}".format(self.subject)
+        return "{} ({})".format(self.subject, self.id)
+
+    def update_global_notification(self, account_notification, **kwargs):
+        """
+        Updates a global notification.
+
+        :calls: `PUT /api/v1/accounts/:account_id/account_notifications/:id \
+        <https://canvas.instructure.com/doc/api/account_notifications.html#method.account_notifications.update>`_
+
+        :param account_notification: The notification to update with.
+        :type account_notification: dict
+
+        :rtype: :class:`canvasapi.account.AccountNotification`
+        """
+        required_key_list = ["subject", "message", "start_at", "end_at"]
+        required_keys_present = all(
+            (x in account_notification for x in required_key_list)
+        )
+
+        if isinstance(account_notification, dict) and required_keys_present:
+            kwargs["account_notification"] = account_notification
+        else:
+            raise RequiredFieldMissing(
+                (
+                    "account_notification must be a dictionary with keys "
+                    "'subject', 'message', 'start_at', and 'end_at'."
+                )
+            )
+
+        response = self._requester.request(
+            "PUT",
+            "accounts/{}/account_notifications/{}".format(self.account_id, self.id),
+            _kwargs=combine_kwargs(**kwargs),
+        )
+
+        return AccountNotification(self._requester, response.json())
 
 
 @python_2_unicode_compatible
 class AccountReport(CanvasObject):
     def __str__(self):  # pragma: no cover
```

Comment:
Remove `# pragma: no cover` and add test

Reply:
Added test.